x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_enrich.yml

```diff
@@ -1,6 +1,8 @@
 ---
 setup:
   - skip:
+      version: " - 7.10.99"
+      reason: "ESQL is available in 7.11+"
```

Comment:
Is this correct? Or should it be 8.11? ( I could be missing how all this actually fits together )